.github/workflows/main.yml

```diff
@@ -60,6 +60,9 @@ jobs:
           - ${{ inputs.java }}
         image_suffix: [python3-ubuntu, ubuntu, r-ubuntu, python3-r-ubuntu]
     steps:
+      - name : Test - Run spark application for standalone cluster on docker
```

Comment:
I was mean `- name: Test - Checkout Spark repository` L166

Reply:
Oh, my mistake. Only search `Checkout Spark repository`. What difference between `Checkout Spark repository` and `Test - Checkout Spark repository` ? A little misleading

Reply:
Maybe rename `Checkout Spark repository` as `Checkout Spark Docker repository`?

Reply:
Yep, feel free to rename it